docs/README.md

```diff
@@ -0,0 +1,66 @@
+<img src="assets/Forge_logo.svg">
+
+MinecraftForge [![Latest Release](https://img.shields.io/maven-metadata/v/https/files.minecraftforge.net/maven/net/minecraftforge/forge/maven-metadata.xml.svg?label=latest)](https://files.minecraftforge.net) [![Discord](https://img.shields.io/discord/313125603924639766.svg?color=%237289da&label=discord&logo=discord&logoColor=%237289da)](https://discord.gg/UvedJ9m) [![Support](https://img.shields.io/badge/patreon-support-orange.svg)](https://www.patreon.com/LexManos)
+=============
+
+Forge is a free, open-source modding API all of your favourite mods use!
+
+| Version  | Support |
+| ------------- | ------------- |
+| 1.14.2  | Development  |
+| 1.13.2  | Maintained  |
+| 1.12.2  | Maintained  |
+
+* [Download]
+* [Forum]
+* [Discord]
+* [Documentation]
+
+#### Notes:
+- Introduced in 1.13 was a new FML, information found [here](NewFML.md).
+
+# Installing Forge
+
+Go to [the Forge website](http://files.minecraftforge.net)
+ and select the minecraft version you wish to get forge for from the list.
```

Comment:
Use 'Minecraft', 'Forge', 'GitHub', etc. throughout, as these are the 'official' names.